packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.html

```diff
@@ -1,5 +1,43 @@
 <p data-translate="telephony_line_tab_beta_softphone_description"></p>
 
+<oui-message
+    data-dismissable
+    data-type="success"
+    data-ng-if="$ctrl.error_email===false && ($ctrl.disableButton)"
```

Comment:
```suggestion
    data-ng-if="!$ctrl.errorEmail && $ctrl.disableButton"
```

Reply:
nope since it will appear if I have no value also, so typing is needed here